tests/manage/mcg/test_nsfs.py

```diff
@@ -24,6 +25,70 @@ class TestNSFSObjectIntegrity(MCGTest):
 
     """
 
+    @pytest.fixture(autouse=True, scope="class")
+    def modify_endpoint_scc(self, request) -> None:
```

Comment:
Move this fixture to `conftest.py` so that any other tests using NSFS (ex: system tests) can make use of this

Reply:
Fixed in Addressed  in 3a8419652df6170cb225428170115076036a3621